matchers/be_below.go

```diff
@@ -2,10 +2,13 @@ package matchers
 
 import "fmt"
 
+// BeBelowMatcher accepts a numerical value. It succeeds if the actual is
+// less than the expected.
```

Comment:
Really the value must be a float64 and not any numeric. This is something we might want to support though.